recipe/meta.yaml

```diff
@@ -6,79 +6,63 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nslsii-{{ version }}.tar.gz
   sha256: fc3e836ce70bee8f578881ae2df525349774951edfc0c1b8ab90ee0c4a777e57
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<37 or osx]
-  ignore_run_exports:
-    - python_abi
+  entry_points:
+    - sync-experiment = nslsii.sync_experiment:main
+    - what-is-ioc = nslsii.epics_utils:main
+  noarch: python
```

Comment:
If we decide to skip Windows builds, this line needs to be replaced with `skip: true  # [win]` and then the feedstock should be rerendered.